docs/version-1.9/modules/en/pages/installation-setup/installation/airgapped-environment.adoc

```diff
@@ -211,67 +106,61 @@ privateRegistry:
     registrySecret: <REGISTRY_SECRET_NAME>
 ----
```

Comment:
`defaultSettings.registrySecret: <SECRET_NAME>`  not exist in `values.yaml`([ref](https://github.com/longhorn/website/pull/1154/files)) Additionally, the example is missing the `privateRegistry.createSecret` setting.